static/app/bootstrap/initializeSdk.tsx

```diff
@@ -51,19 +50,17 @@ const shouldOverrideBrowserProfiling = window?.__initialData?.user?.isSuperuser;
  */
 function getSentryIntegrations(routes?: Function) {
   const integrations = [
-    extraErrorDataIntegration({
+    Sentry.extraErrorDataIntegration({
       // 6 is arbitrary, seems like a nice number
       depth: 6,
     }),
-    Sentry.metrics.metricsAggregatorIntegration(),
```

Comment:
@AbhiPrasad just to be clear, the migration here is to just remove this, right?

Reply:
yup, no longer needed!